packages/plugin-linebreak/LineBreak.ts

```diff
@@ -29,8 +36,13 @@ export class LineBreak extends JWPlugin {
                 // TODO: do this less naively to account for formatting space.
                 context.currentNode = node.nextSibling;
                 domNodes.push(node.nextSibling);
+                const siblingAttributes = Parser.parseAttributes(node.nextSibling);
+                siblingAttributes.forEach((value: string, name: string) => {
+                    attributes.set(name, value);
+                });
```

Comment:
Same here, why move the attributes from a node to another ? This second BR is somewhat "fake". Remember old versions of the editor where there were the "virtuals" ? You wouldn't want to handle their attributes, would you ?